src/pages/en/tutorial/2-astro-pages/1.md

```diff
@@ -0,0 +1,153 @@
+---
+layout: ~/layouts/TutorialLayout.astro
+title: Creating new pages
+setup: |
+  import Checklist from '~/components/Checklist.astro';
+  import Goals from '~/components/tutorial/Goals.astro';
+---
+
+<Goals>
+  - added two new pages to your website by creating two new `.astro` files
+  - added navigation links to access each page of your site from all pages
+  - committed your changes to GitHub
+  - deployed a new version of your website at your `.netlify.app` web address
+</Goals>
+
+Now that you know what has to happen to create a new page on your website, let's do it!
+
+## Create a new `.astro` file
+
+1. In the files pane of your code editor, navigate to the filepath `src/pages/` where you will see the existing file `index.astro`
+
+2. In that same folder, create a new file named `about.astro`.
+
+3. Copy, or retype the contents of `index.astro` into your new `about.astro` file.
+
+    :::note
+    Your editor might show a solid white circle on the tab label for this file. This means that the file is not yet saved. Under the File menu, enable "Auto Save" and you should no longer need to save any files manually.
```

Comment:
I'm saying "code editor" in general sometimes because that would be appropriate for both VS Code and StackBlitz. I think as the tutorial goes on, I tend to use "local-oriented" terms more, expecting that the StackBlitz people will have caught on by now.